Added note for problem with draining nodes and S2D

Paxilein opened the pull request:
Added note for problem with draining nodes and S2D in azure-stack-replace-component.md

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Paxilein committed:
Added note for problem with draining nodes and S2D

Added note for problem with draining nodes and S2D in azure-stack-replace-component.md

GitHubber17 committed:
Update azure-stack-replace-component.md
